SROS2_Linux.md

```diff
@@ -85,13 +85,12 @@ These variables need to be defined in each terminal used for the demo. For conve
 
 ### Run the demo
 
-ROS2 allows you to [change DDS implementation at runtime](https://docs.ros.org/en/rolling/Guides/Working-with-multiple-RMW-implementations.html).
-This demo can be run with fastrtps by setting:
-```bash
-export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
-```
-And with Connext by setting:
+ROS 2 allows you to [change DDS implementation at runtime](https://docs.ros.org/en/rolling/Guides/Working-with-multiple-RMW-implementations.html).
+This demo can be run with FastDDS / CycloneDDS / ConnextDDS by setting the `RMW_IMPLEMENTATION` variable, e.g.:
+
 ```bash
+export RMW_IMPLEMENTATION=rmw_fastrtps_cpp  # or
+export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp  # or
 export RMW_IMPLEMENTATION=rmw_connextdds
```

Comment:
security does not even work...

```bash
root@51cdd59e1f3e:~# export RMW_IMPLEMENTATION=rmw_connextdds
root@51cdd59e1f3e:~# export ROS_SECURITY_KEYSTORE=~/sros2_demo/demo_keystore
root@51cdd59e1f3e:~# export ROS_SECURITY_ENABLE=true
root@51cdd59e1f3e:~# export ROS_SECURITY_STRATEGY=Enforce
root@51cdd59e1f3e:~# ros2 run demo_nodes_cpp talker --ros-args --enclave /talker_listener/talker
[INFO] [1715030503.075598598] [rcl]: Found security directory: /root/sros2_demo/demo_keystore/enclaves/talker_listener/talker
RTI Connext DDS Non-commercial license is for academic, research, evaluation and personal use only. USE FOR COMMERCIAL PURPOSES IS PROHIBITED. See RTI_LICENSE.TXT for terms. Download free tools at rti.com/ncl. License issued to Non-Commercial User license@rti.com For non-production use only.
Expires on 00-jan-00 See www.rti.com for more information.
[CREATE Participant] RTIOsapiLibrary_openEx:!open library=libnddssecurityd.so: libnddssecurityd.so: cannot open shared object file: No such file or directory
[CREATE Participant] DDS_DomainParticipantTrustPlugins_initialize:!failed to load library
[CREATE Participant] DDS_DomainParticipant_createI:!create builtin trust plugins support
[CREATE Participant] DDS_DomainParticipantFactory_create_participant_disabledI:!create participant
[ERROR] [1715030503.205463802] [rmw_connextdds]: failed to create DDS participant
[ERROR] [1715030503.205703150] [rmw_connextdds]: failed to initialize DomainParticipant
[ERROR] [1715030503.205735098] [rmw_connextdds]: failed to initialize node in context
terminate called after throwing an instance of 'rclcpp::exceptions::RCLError'
  what():  failed to initialize rcl node: failed to create DDS participant, at ./src/common/rmw_context.cpp:557, at ./src/rcl/node.c:252
[ros2run]: Aborted
```

Reply:
hmmm wondering for how long this doesnt work :thinking: I don't have a license ATM but will give it a shot sometimes this week

Reply:
> libnddssecurityd.so: cannot open shared object file: No such file or directory

@fujitatomoya are you sure you installed the Connext Security plugins on your system ?
It looks like the relevant libraries are missing

Could you successfully complete this section of the tutorial? https://github.com/ros2/sros2/blob/rolling/SROS2_Linux.md#additional-configuration-for-rti-connext

Reply:
ah that is i did not do, sorry. i need to take this back.